WordPress/src/main/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsViewModel.kt

```diff
@@ -2,16 +2,19 @@ package org.wordpress.android.ui.reader.discover.interests
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
-import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.launch
 import org.wordpress.android.models.ReaderTagList
+import org.wordpress.android.modules.UI_THREAD
 import org.wordpress.android.ui.reader.repository.ReaderTagRepository
+import org.wordpress.android.viewmodel.ScopedViewModel
 import javax.inject.Inject
+import javax.inject.Named
 
 class ReaderInterestsViewModel @Inject constructor(
+    @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher,
     private val readerTagRepository: ReaderTagRepository
-) : ViewModel() {
+) : ScopedViewModel(mainDispatcher) {
```

Comment:
Used `ScopedViewModel` here for consistency. Also tried out `viewModelScope` [in this branch](https://github.com/wordpress-mobile/WordPress-Android/tree/issue/12026-ri-show-interests-chips-viewmodelscope)  as our `viewmodel` libraries are upgraded and support it. I'm fine with both approaches. Let me know whatever works :).

Reply:
I'm fine with both approaches as well ;). Whatever you prefer.

Reply:
Swapped `ScopedViewModel` with `viewModelScope` in 29e7481.